### PR TITLE
Updated server constants for better performance

### PR DESCRIPTION
**Pull request by lintongj**
Updated server constants, e.g., resyncperiod, QPS, burst, etc., for better performance

Signed-off-by: Lintong Jiang <lintongj@vmware.com>

### CODE REVIEW

File: pkg/cmd/datamgr/cli/server/server.go [hunk 1]
```diff
@@ -287,7 +283,6 @@ func newServer(f client.Factory, config serverConfig, logger *logrus.Logger) (*s
 		namespace:             f.Namespace(),
 		metricsAddress:        config.metricsAddress,
 		kubeClient:            kubeClient,
-		veleroClient:          veleroClient,
```
Comment (deepakkinni) [inline pkg/cmd/datamgr/cli/server/server.go:290]: no users of veleroClient?

Comment (lintongj) [reply pkg/cmd/datamgr/cli/server/server.go:290]: Yes, veleroClient is unused in data-manager. And it should not be used.

**Review by deepakkinni — APPROVED** [review]
lgtm

**Review by xing-yang — APPROVED** [review]
LGTM

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/387/

**Pull request merged**